examples/n2c-miniprotocols/src/main.rs

```diff
@@ -42,20 +42,54 @@ async fn do_chainsync(client: &mut NodeClient) {
         };
     }
 }
+async fn do_tx_monitor(client: &mut NodeClient) {
+    loop {
+        match client.txmonitor().state(){
+            txmonitor::State::Idle => {
+                info!("state is idle ");
+
+                let acquire_mempool = client.txmonitor().acquire().await.unwrap();
+                info!("acquireMempool: {:?}", acquire_mempool);
+            },
+            txmonitor::State::Acquiring => {
+                info!("state is acquiring ");
+            },
+            txmonitor::State::Acquired => {
+                info!("state is acquired ");
+
+                let size_and_capacity = client.txmonitor().query_size_and_capacity().await.unwrap();
+                info!("tx in mempool: {:?}", size_and_capacity.number_of_txs);
+
+                while let Some(tx) = client.txmonitor().query_next_tx().await.unwrap() {
+                    info!("nextTransactionResponse: {:?}", tx);
+                }
+                let re_acquire_mempool = client.txmonitor().acquire().await.unwrap();
+                info!("acquireMempool: {:?}", re_acquire_mempool);
+
+            },
+            txmonitor::State::Busy => {
+                info!("state is busy ");
+            },
+            txmonitor::State::Done => {
+                info!("state is done ");
+            },
+        }
+    }
+}
 
 #[cfg(target_family = "unix")]
 #[tokio::main]
 async fn main() {
     tracing::subscriber::set_global_default(
         tracing_subscriber::FmtSubscriber::builder()
-            .with_max_level(tracing::Level::TRACE)
+            .with_max_level(tracing::Level::INFO)
             .finish(),
     )
     .unwrap();
 
     // we connect to the unix socket of the local node. Make sure you have the right
     // path for your environment
-    let socket_path = "/tmp/node.socket";
+    let socket_path = "/home/kalomaaantokyo/cardano-my-node/db/socket";
```

Comment:
Perhaps just read this from an env var?

Reply:
@kalomaaan an env var as @rvcas suggests would be great, but if not possible in this PR, lets keep the previous convention for the socket path